framework/source/class/qx/ui/virtual/selection/MModel.js

```diff
@@ -46,14 +46,16 @@ qx.Mixin.define("qx.ui.virtual.selection.MModel",
   construct : function()
   {
     this._initSelectionManager();
-    this.__defaultSelection = new qx.data.Array();
-    this.initSelection(this.__defaultSelection);
+    this.__selection = new qx.data.Array();
+    this.__selection.addListener("change", this._onChangeSelection, this); 
+    this._applySelection(this.__selection, null);
   },
 
 
   properties :
   {
     /** Current selected items */
+    /* psuedo property - implemented manually in code below
     selection :
```

Comment:
Why is the property then needed at all?
Shouldn't this completely be removed as the automatic created getters and setters will else fight with the manual created ones?

Reply:
no, that's why it's commented out - the selection property is needed, but the property mechanism isn't quite good enough so the property has to be implemented with hand crafted functions and events.  When I have to do this, I tend to put in a commented-out property and mark it as psuedo, just so that there is some kind of consistent documentation

Reply:
Oh! I didn't notice that it was commented out.